rfd/0016-dynamic-configuration.md

```diff
@@ -180,33 +180,32 @@ auth server initialization procedure:
 
 ## Implementation
 
-The resource label `origin` is to be used as the key indicator when determining
-the configuration sources and their precedence.
+The resource label `teleport.io/origin` (below shortened to `origin` for
```

Comment:
the only nit is that we don't actually have teleport.io domain, we do have `teleport.dev` and `teleport.sh` domain, so if someone thinks it's a real domain name, you might want to use those ones

Reply:
I mean, I'm completely indifferent in this regard. Somebody else has to decide which prefix is the most appropriate :)

Reply:
I think `teleport.dev` makes sense to use, it redirects to our website and we're not running a service on it.

Reply:
FWIW, it need not be a domain. Mere `teleport/origin` may suffice.

Reply:
Domain name makes sense if we expect 3rd party vendors to integrate with Teleport in the future.
For example, if Terraform was creating Teleport resources, it could set `terraform.io/foo: bar` labels for its own use and have a clear namespace.

`teleport.dev` or `goteleport.com` should be fine.

Reply:
agreed on teleport.dev